mars/dataframe/base/get_dummies.py

```diff
@@ -0,0 +1,283 @@
+# Copyright 1999-2021 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import pandas as pd
+
+from ...core import OutputType, recursive_tile
+from ...serialization.serializables import StringField, BoolField, AnyField, ListField
+from ..core import SERIES_TYPE
+from ..datasource.dataframe import from_pandas as from_pandas_df
+from ..datasource.series import from_pandas as from_pandas_series
+from ..initializer import Series as asseries
+from ..operands import DataFrameOperand, DataFrameOperandMixin
+from ..reduction.unique import unique
+
+_encoding_dtype_kind = ['O', 'S', 'U']
+
+
+class DataFrameGetDummies(DataFrameOperand, DataFrameOperandMixin):
+    prefix = AnyField('prefix')
+    prefix_sep = StringField('prefix_sep')
+    dummy_na = BoolField('dummy_na')
+    columns = ListField('columns')
+    sparse = BoolField('sparse')
+    drop_first = BoolField('drop_first')
+    dtype = AnyField('dtype')
+
+    def __init__(self, prefix=None, prefix_sep="_", dummy_na=False,
+                 columns=None, sparse=False, drop_first=False, dtype=None, **kws):
+        super().__init__(prefix=prefix, prefix_sep=prefix_sep, dummy_na=dummy_na,
+                         columns=columns, sparse=sparse, drop_first=drop_first, dtype=dtype, **kws)
+        self.output_types = [OutputType.dataframe]
+
+    @classmethod
+    def tile(cls, op):
+        inp = op.inputs[0]
+        out = op.outputs[0]
+        if len(inp.chunks) == 1 or isinstance(inp, SERIES_TYPE):
```

Comment:
When `inp` is a series and has multi chunks, the logic should be different.

Reply:
`chunk = chunk_op.new_chunk(inp.chunks, kws=[chunk_param])` this line means all input chunks data will be in one worker which is danger for huge series.